packages/plugin-core/src/abTests.ts

```diff
@@ -122,6 +108,9 @@ export const CURRENT_AB_TESTS = [
   UPGRADE_TOAST_WORDING_TEST,
   SELF_CONTAINED_VAULTS_TEST,
   MEETING_NOTE_TUTORIAL_TEST,
-  MEETING_NOTE_FEATURE_SHOWCASE_TEST,
   GRAPH_THEME_TEST,
 ];
+
+// Concluded Experiments (NOTE: make sure that you don't re-use these strings as
+// the name of a future AB Test to avoid conflicting strings):
+// - MeetingNoteFeatureShowcaseTest
```

Comment:
can we have this in an SOP instead of the code? if we introduce these changes: https://github.com/dendronhq/org-private/commit/6d558b90ac2516861dab27d1c5d92f3e21a91b05

we can find all previous tests by filtering via [[abtest|dendron://private/ext.airtable.table.events#abtest]]

Reply:
actually, thinking about this some more -should ab test have a {year}-{month} prefix so we never have to worry about this?

Reply:
yeah that's a good idea. so for this one, `2022-04-MeetingNoteFeatureShowcase`